content/hi/docs/tasks/tools/install-kubectl-macos.md

```diff
@@ -0,0 +1,248 @@
+---
+
+title: macOS पर kubectl इंस्टॉल और सेट करें
+content_type: task
+weight: 10
+card:
+  name: tasks
+  weight: 20
+  title: macOS पर kubectl इंस्टॉल करें
+---
+
+## {{% heading "पूर्वाकांक्षित" %}}
+
+आपको kubectl संस्करण का उपयोग करना चाहिए जो आपके क्लस्टर के एक मामूली संस्करण अंतर के भीतर हो. उदाहरण के लिए,  v{{< skew latestVersion >}} क्लाइंट निम्नलिखित के साथ संवाद कर सकते हैं v{{< skew prevMinorVersion >}}, v{{< skew latestVersion >}}, and v{{< skew nextMinorVersion >}} कण्ट्रोल प्लेन.
+kubectl के नवीनतम संस्करण का उपयोग करने से अप्रत्याशित मुद्दों से बचने में मदद मिलती है.
```

Comment:
```suggestion
kubectl के नए संस्करण का उपयोग करने से समस्या से बचत हो सकती है.
```

Reply:
@anubha-v-ardhan Should we use `नए संस्करण`? It will change the meaning a bit, but it makes sentence more easy to read.

Reply:
I think `नवीनतम` is not a complex word to use. Is it?

Reply:
@anubha-v-ardhan just let me know which changes needs to be resolved. I will resolve those.